src/app/shared/services/query.service.ts

```diff
@@ -12,14 +12,16 @@ export class QueryService {
   private url = `${environment.baseUrl}/api/multieditor`;
   private schemaUrl = `${environment.baseUrl}/schemas/records`;
 
-  constructor(private http: Http) {}
+  constructor(private http: Http) { }
 
-  save(userActions: UserActions, checkedRecords: string[]): Promise<void> {
+  save(userActions: UserActions, checkedRecords: string[], allSelected: boolean): Promise<void> {
```

Comment:
also not necessary to have allSelected anymore.

Reply:
I still need to send it to the backend in the case that every record is selected.